lib/aur-repo

```diff
@@ -8,6 +8,9 @@ PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 modifier=local table_format=0
 
 db_table() {
+    # do not extract an empty database (#727)
+    (( $(bsdtar -tf "$2" | wc -l) )) || return
+
     bsdtar -Oxf "$2" '*/desc' | awk -v table="$1" '
```

Comment:
Maybe we should put the output in a tempfile and only pass it to awk if it is non-empty. 

The advantage being that in case of failure we would at keep the input that caused the error.

Reply:
awk can be used to work with both empty and non-empty inputs

Reply:
Yeah, the issue here is with `bsdtar` erroring out if the archive is empty; awk would work regardless.